wdl-format/src/element.rs

```diff
@@ -0,0 +1,508 @@
+//! Elements used during formatting.
+
+use std::collections::HashMap;
+use std::iter::Peekable;
+
+use nonempty::NonEmpty;
+use wdl_ast::AstToken as _;
+use wdl_ast::Element;
+use wdl_ast::Node;
+use wdl_ast::SyntaxKind;
+
+use crate::NEWLINE;
+use crate::PreToken;
+use crate::TokenStream;
+use crate::Writable;
+
+pub mod node;
+
+/// Trivia associated with some more formidable element.
+///
+/// Trivia would be things like comments and whitespace.
```

Comment:
@peterhuene isn't "trivia" well-defined as being comments and whitespace? Is there any reason to hedge our bets with this language?

Reply:
My thought on this is the same as above—today, what you say is correct, but I don't know that it's necessarily _always_ going to be the case that trivia only includes whitespace and comments.